dotnet/src/Connectors/Connectors.AI.OpenAI/Connectors.AI.OpenAI.csproj

```diff
@@ -20,7 +20,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\SemanticKernel.Abstractions\SemanticKernel.Abstractions.csproj" />
+    <ProjectReference Include="..\..\SemanticKernel\SemanticKernel.csproj" />
```

Comment:
Yeah, it's required to create extension methods for the KernelBuilder class, which is a concrete class in the Core project. 
I'd like to figure out an abstraction that relaxes this requirement, but I'm conflicted as I don't think we should have an IKernelBuilder interface.

Reply:
Agree. Ideally, we should keep Connectors and Kernel projects unaware of each other. 
The fastest way, but not the most elegant one, would be to create  IKernelBuilder interface and put it to the Kernel.Abstraction project. It will allow all the other project add extensions methods to the KernelBuilder witout referencing the Kernel project. 
Alternatively, we could consider having a project/package that knows about both Kernel and 
 Connectors projects and contains all the extension methods.